pyfr/plugins/__init__.py

```diff
@@ -1,4 +1,4 @@
-from pyfr.plugins.base import BasePlugin
+from pyfr.plugins.base import BasePlugin, CLIBasePlugin
```

Comment:
BaseCLIPlugin (may want to rename BasePlugin to something different, but check with @pv101 as he has similar changes in this direction which his invasive turbulent inflow plugin).

Reply:
I checked with @pv101 and he didn't have anything in mind. I reckon we crack on and can just change it later.